File: src/ROT13Test.cs
using NUnit.Framework;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Rot13Challenge.src
{
    [TestFixture]
    public class ROT13Test : ROT13
    {
        [Test]
        public void EncryptTest()
        {
            Assert.AreEqual("Uryyb Jbeyq!", Encrypt("Hello World!"));
        }
    }
}

File: Tests/ROT13Test.vb
Imports NUnit.Framework

<TestFixture()>
Public Class ROT13Test
    Inherits ROT13

    <Test()>
    Public Sub EncryptTest()
        Assert.AreEqual("Uryyb Jbeyq!", Encrypt("Hello World!"), "Unexpected encryption output")
    End Sub
End Class
